backend/app/routers/chat.py

```
@@ -346,40 +345,105 @@ def _maybe_create_alert_from_nl(nl: str, parsed: Dict[str, Any]) -> Optional[Dic
+    m_cd = re.search(r"\bcooldown\s+(?:of\s+)?(\d+(?:\.\d+)?)\s*(sec|s|minute|m|hour|h)\b", low)
+    if m_cd:
+        try:
+            val = float(m_cd.group(1))
+            unit = m_cd.group(2)
+            if unit.startswith("m"):
+                val *= 60.0
+            elif unit.startswith("h"):
+                val *= 3600.0
+            cooldown_sec = val
+        except Exception:
+            pass
```

Ruff (0.15.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)